client/src/app/components/stock-stats/stock-stats.component.html

```diff
@@ -0,0 +1,57 @@
+<mat-card>
+  <div class="stats-header">
+    <h2 class="title">Stats</h2>
+    <mat-icon>trending_up</mat-icon>
+  </div>
+
+  <div class="stats-container">
+    <div class="col-stat">
+      <div class="row-stat">
+        <div class="tooltip">
+          <span class="tooltiptext"> Open definition</span>
+          <span class="stat-label">Open </span>
+        </div>
+        <span id="statOpen">{{ open }}</span>
+      </div>
+      <mat-divider [inset]="true"></mat-divider>
+      <div class="row-stat">
+        <span class="stat-label">High </span>
+        <span id="statHigh">{{ high }}</span>
```

Comment:
refactor: All the ids in css should not be camel case but in the format: 'stat-high'